### DISCUSSION TITLE
Xiaomi Vacuum Local support

### DISCUSSION THREAD

**Discussion by p-rintz** [Tasshack/dreame-vacuum#802] [General]
Hey there,

from what I understand, the Xiaomi Vacuums should support local-only connection if they were setup with the Xiaomi Home App, right?

Does the map work local-only as well by now? The last answer I found was from 2023 where it said map was not supported locally, but elswhere it said it was reverse-engineered?

I am specifically thinking about buying two X20+ Xiaomi Vacuums, but I would want to control them locally.

Thanks!

**Comment by Tasshack** [comment]
Map data is only available via cloud connection but latest vacuums does not even has their local api enabled so no local connection on newer devices.
You should not buy a Dreame device if you don't like the idea of cloud.

**Reply by p-rintz** [reply]
Thanks. Thats too bad.

**Reply by Tasshack** [reply]
Why do you think they are selling it so cheaply compared to other vacuum brands...
